src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private static final String HEAD_FILE = "operation,fruit,quantity";
+    private static final String INVALID_BALANCE_STRING = "b,apple";
+    private static final int EXPECTED_LIST_SIZE = 1;
+    private static final String APPLE = "apple";
+    private static final String ORANGE = "orange";
+    private static final String VALID_BALANCE_APPLE = "b,apple,10";
+    private static final String VALID_SUPPLY_ORANGE = "s,orange,5";
+    private static final int APPLE_QUANTITY = 10;
+    private static final int ORANGE_QUANTITY = 5;
+    private static ParserServiceImpl parser;
+    private List<FruitTransaction> result;
+
+    @BeforeAll
+    public static void beforeClass() {
+        parser = new ParserServiceImpl();
+    }
+
+    @AfterEach
+    public void clearResultList() {
+        result.clear();
+    }
+
+    @Test
+    public void test_parse_with_null_input_ok() {
```

Comment:
Please, fix test naming in every case [rule](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#remember-about-naming-of-test-methods)